R2.3 Clarify the parenx API

anisotropi4 opened the pull request:
R2.3 Clarify the parenx API; address unusual CLI model and skeletonize.py/voronoi.py naming

Robinlovelace commented:
Failing on actions with

ModuleNotFoundError: No module named 'readline'

And 

! ImportError: cannot import name 'get_nx' from 'parenx.shared' (/home/runner/.virtualenvs/r-reticulate/lib/python3.12/site-packages/parenx/shared.py)

See https://github.com/nptscot/networkmerge/actions/runs/16532420509/job/46760580706?pr=143#step:7:749 for more.

Can you investigate @anisotropi4 ? Thanks

anisotropi4 commented:
@Robinlovelace the error was because I changed the name of the `get_nx` to `get_primal` in the `parenx`package. I have now updated the Quarto files to use the new function name. Hopefully that resolves the issue.
